biogtr/datasets/base_dataset.py

```diff
@@ -27,7 +28,9 @@ def __init__(
         """Initialize Dataset.
 
         Args:
-            files: a list of files, file types are combined in subclasses
+            label_files: a list of paths to label files.
+            should at least contain detections for inference, detections + tracks for training.
```

Comment:
Indent second line of comments when referring to the same field!

```suggestion
                should at least contain detections for inference, detections + tracks for training.
```